src/styles/global.css

```diff
@@ -0,0 +1,31 @@
+/*Navbar Component*/
+
+#Navbar {
+    width: 100%;
+    background-color: #333333;
+}
+
+#LinkList {
```

Comment:
You almost never want to use ids when styling. Incredibly difficult to override that level of specificity.